src/features/profile/components/templates/Profile.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import CreatedRoadmapList from '../atoms/MadeRoadmapList';
+import { ProfileBio } from '../atoms/ProfileBio';
+import ProfileHeader from '../molecules/ProfileHeader';
+import ProfileSecondBox from '../organisms/ProfileSecondBox';
+import ProfileThirdBox from '../organisms/ProfileThirdBox';
+export default function Profile() {
+  return (
+    <div className="flex w-[950px] flex-col gap-12">
+      {ProfileHeader()}
```

Comment:
```suggestion
      <ProfileHeader />
```

올바른 컴포넌트 호출 방식을 사용해주세요